src/dev/konnector.md

```diff
@@ -2,110 +2,376 @@
 
 ## Introduction
 
-A connector is a simple script which imports data from other web services and put it in your cozy.
-Each connector is an independant application, managed by the [Cozy Collect]
-application.
+A connector (as know as _konnector_) is a script that imports data from another web service and put those data into your cozy.
```

Comment:
s/as/also
s/those/that

Reply:
TIL I always thought that _a.k.a._ stands for _as known as_

Reply:
According to some references, _data_ is a plural name and should be treated as such.

- https://forum.wordreference.com/threads/these-data-this-data.3271297/
- http://grammarist.com/usage/data/